File: pruebasProyecto/person.cpp
#include "person.h"
using namespace std;

// Constructor de la clase Person
Person::Person(int _id, const string& _name, const string& _lastName,
    const string& _birthdate, const string& _gender,
    const string& _address, int _phoneNumber, const string& _email) 
    : id(_id), name(_name), lastName(_lastName), birthdate(_birthdate),
      gender(_gender), address(_address), phoneNumber(_phoneNumber), email(_email) {
}

// Constructor vacío de la clase Person
Person::Person(){}

// Métodos get para obtener los valores de los atributos
int Person::getId(){ return id; }
string Person::getName(){ return name; }
string Person::getLastName(){ return lastName; }
string Person::getBirthdate()  { return birthdate; }
string Person::getGender(){ return gender; }
string Person::getAddress(){ return address; }
int Person::getPhoneNumber(){ return phoneNumber; }
string Person::getEmail(){ return email; }    

// Métodos set para modificar los valores de los atributos
void Person::setId(int _id) { id = _id; }
void Person::setName(const string& _name) { name = _name; }
void Person::setLastName(const string& _lastName) { lastName = _lastName; }
void Person::setBirthdate(const string& _birthdate) { birthdate = _birthdate; }
void Person::setGender(const string& _gender) { gender = _gender; }
void Person::setAddress(const string& _address) { address = _address; }
void Person::setPhoneNumber(int _phoneNumber) { phoneNumber = _phoneNumber; }
void Person::setEmail(const string& _email) { email = _email; }

File: pythonPr/person.py

class Person:
    def __init__(self, _ident = None, _name = None, _lastName = None, _birthdate = None, _gender = None, _address = None, _phoneNumber = None, _email = None):
        self.ident = _ident
        self.name = _name
        self.lastName = _lastName
        self.birthdate = _birthdate
        self.gender = _gender
        self.address = _address
        self.phoneNumber = _phoneNumber
        self.email = _email

    # Métodos get para obtener los valores de los atributos
    def get_ident(self):
        return self.ident

    def getName(self):
        return self.name

    def getLastName(self):
        return self.lastName

    def getBirthdate(self):
        return self.birthdate

    def getGender(self):
        return self.gender

    def getAddress(self):
        return self.address

    def getPhoneNumber(self):
        return self.phoneNumber

    def getEmail(self):
        return self.email

    # Métodos set para modificar los valores de los atributos
    def setId(self, _ident):
        self.ident = _ident

    def setName(self, _name):
        self.name = _name

    def setLastName(self, _lastName):
        self.lastName = _lastName

    def setBirthdate(self, _birthdate):
        self.birthdate = _birthdate

    def setGender(self, _gender):
        self.gender = _gender

    def setAddress(self, _address):
        self.address = _address

    def setPhoneNumber(self, _phoneNumber):
        self.phoneNumber = _phoneNumber

    def setEmail(self, _email):
        self.email = _email